cpp/daal/src/externals/service_stat_mkl.h

```diff
@@ -130,17 +136,25 @@ extern "C"
 
     static void _daal_mkl_threader_for(DAAL_INT n, DAAL_INT threads_request, void * a, func_type func)
     {
-        fpk_vsl_serv_threader_for(n, threads_request, a, func);
+        // // mkl_vsl_serv_threader_for(n, threads_request, a, func);
+        for (DAAL_INT i = 0; i < n; i++)
+        {
+            func(i, 0, 1, a);
+        }
     }
 
     static void _daal_mkl_threader_for_ordered(DAAL_INT n, DAAL_INT threads_request, void * a, func_type func)
     {
-        fpk_vsl_serv_threader_for_ordered(n, threads_request, a, func);
+        // mkl_vsl_serv_threader_for_ordered(n, threads_request, a, func);
+        for (DAAL_INT i = 0; i < n; i++)
```

Comment:
do I understand correctly that parallel for is replaced by usual one here? if so, does it affect performance?